docs/store/datastore.md

```diff
@@ -110,6 +112,14 @@ may contain the contents of this file. If configured in the function pod, MLRun
 and points `GOOGLE_APPLICATION_CREDENTIALS` at it. An exception is `BigQuerySource`, which passes `GCP_CREDENTIALS`'s
 contents directly to the query engine.
 
+### Databricks file system
+```{Admonition} Note
+Spark ingestion does not support DBFS.
```

Comment:
I think this line can be removed - but if not say: Not supported by runtimes "spark" and "remote-spark"